README.md

```diff
@@ -1 +1,31 @@
-# Etch-a-sketch-Bashaer
\ No newline at end of file
+# Etch-a-sketch-Bashaer
+## :memo: How do I vision my webpage to be like?
+![pixilart-drawing](https://user-images.githubusercontent.com/70339752/98022490-5b981100-1e0e-11eb-8a7a-e0b8e473b1f1.png)
```

Comment:
the size of this pix is kinda small, try to wrap the URL into an <img> tag to adjust the photo height& width :sunglasses:

Reply:
That's right ,thank you ! Just updated it : )